src/components/Logo/index.js

```diff
@@ -1,10 +1,25 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 import configuration from 'shared/services/configuration/configuration';
 
 export const Logo = props => (
-  <a href={configuration.links.home}>
-    <img alt="Logo" src={configuration.logoUrl} height={configuration.logoHeight} {...props} />
+  <a href={props.tall ? configuration.links.home : '/'}>
```

Comment:
It looks that on small devices in the incident form (not authenticated) the logo is not showing correct yet. Cen we fix that as well?

Reply:
I had a short call with @aditudorache about this, I will approve and merge this as it is since it blocks `Weesp` @aditudorache said he will contact you (@boris-arkenaar) about his comments ;)

Reply:
Yes. I will try and fix this in a new PR.

Reply:
@aditudorache The logo looks fine to me on small devices. Could you indicate what's wrong?

Reply:
I believe is fixed already in another branch. Let's park it for now!

Reply:
Cool, thanks!